content/news/2023-10-21-bevy-0.12/index.md

```diff
@@ -19,56 +19,100 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 <div class="release-feature-authors">authors: @author</div>
 
-## Automatic Batching and Instancing, and the Road to GPU-driven Rendering
+## Automatic Batching/Instancing of Draw Commands
 
-<div class="release-feature-authors">authors: Rob Swain (@superdump), @james-j-obrien, @JMS55, @inodentry, @robtfm, @nicopap, @teoxoy, @IceSentry, @Elabajaba</div>
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
 
-Bevy's renderer performance for 2D and 3D meshes can improve a lot. There are bottlenecks on both the CPU and GPU side, which can be lessened to give significantly higher frame rates. As always with Bevy, we want to make the most of the platforms you use, from the constraints of WebGL2 and mobile devices, to the highest-end native discrete graphics cards. A solid foundation can support all of this.
+**Bevy 0.12** now automatically batches/instances draw commands where possible. This cuts down the number of draw calls, which yields significant performance wins!
+
+This required a number of architectural changes, including how we store and access per-entity mesh data (more on this later).
+
+Here are some benches of the old unbatched approach (0.11) to the new batched approach (0.12):
+
+### 2D Mesh Bevymark (frames per second, more is better)
+
+This renders 160,000 entities with textured quad meshes (160 groups of 1,000 entities each, each group sharing a material). This means we can batch each group, resulting in only 160 instanced draw calls when batching is enabled.
```

Comment:
This is misleading - as the batch size for the graph below was set to 1, it will do 160k draws I think.